reference/examples/TransferTokensUsingPublicAddress.md

```diff
@@ -0,0 +1,13 @@
+
+Transfer HBAR or tokens to a Hedera account using their public-address.
+
+Reference: HIP-583
+
+## Example 1
+- Create a Ethereum Account Address / public-address	(Example: 0xb794f5ea0ba39494ce839613fffba74279579268)
+- Transfer tokens using the `TransferTransaction` to the Etherum Account Address
+- Transfer tokens using the `TransferTransaction` to the long zero form the of the account ID
+- Get the child receipt or child record to return the Hedera Account ID for both of the accounts that were created
+- Show that the created accounts do not have an account public key by requesting the account info
+- Use the newly created account as the transaction fee payer to pay for a transfer transaction
```

Comment:
What account/key/address is signing the transfer transaction so that the public key can be extracted and applied?

Reply:
Assuming a valid hollow account it would be the public key that maps to the evm address